ad-gen.coffee

```diff
@@ -71,10 +71,24 @@ pageLevelCodePart1 = "
 "
 
 pageLevelCodePart2 = "
+  function enforceGTPCCPA(){
+    if(! window.__uspapi ){ return; }
+    window.__uspapi('getUSPData', 2,
+      function( data, success ){
+        if( !success ){ return; }
+        if( data.uspString.match(/^2.Y/gi) ){
```

Comment:
We've used the following pattern in our CMP, which may be a little clearer?

```suggestion
        if( data.uspString.charAt(2) === 'Y' ){
```

Reply:
👍